src/CMakeLists.txt

```diff
@@ -170,6 +170,9 @@ if (${AMBER_ENABLE_TESTS})
   target_include_directories(amber_unittests PRIVATE
       ${gmock_SOURCE_DIR}/include)
   target_link_libraries(amber_unittests libamber gmock_main)
+  if (${AMBER_ENABLE_SHADERC})
+    target_link_libraries(amber_unittests shaderc SPIRV)
```

Comment:
I wonder how this worked before ....

Reply:
Hm.. I don't actually need this.
I thought I did so I could access the shaderc/env.h header.  But I don't!  Will remove

Reply:
Removed